Fix hdf, cd32 and adf in (Disc 1 of x) format

Voljega opened the pull request:
- Fix CD32 isos/cues launching
- Fix multi adf floppies loading for main (Disc 1 of x) format
- Add hdf compatibility

nadenislamarre commented:
# disable port 2 (otherwise, the joystick goes on it)
        commandArray.append("-s")
        commandArray.append("joyport2=")

not a problem to remove this code ?
i added it otherwise we got issues with pads.

Voljega commented:
> 
> 
> ```
>     # disable port 2 (otherwise, the joystick goes on it)
>     commandArray.append("-s")
>     commandArray.append("joyport2=")
> ```
> 
> not a problem to remove this code ?
> i added it otherwise we got issues with pads.

I did'nt remove it, I moved it below, the diff is doing some strange things, but if you looked at the raw file, I didn't touched anything regarding controllers at all, just moved the block
